README.md

```diff
@@ -371,6 +376,87 @@ Failed: text 'foo' not present on element with locator 'By(css selector, h1)'.
+
+All changes below is for the library focus only on providing an easy way for software engineers to create robust end-to-end tests.
+
+### Functions that will be removed
+
+1. [getBodyElementFromCurrentBrowserOrBrowserInstance](#getBodyElementFromCurrentBrowserOrBrowserInstance)
+2. [openNewBrowserInTheSamePage](#openNewBrowserInTheSamePage)
+
+#### What to do?
+
+- Remove all functions above to don't have trouble on next major version (v4.0.0).
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
+
+```
+Protractor-helper warning: Function 'getBodyElementFromCurrentBrowserOrBrowserInstance' will be deprecated in version 4.0.0! [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]
+```
+
+### Functions that will be replaced
+
+The 8 below will be replaced to reduce complexity, with the new functions with less arguments and smaller name.
+
+| Old                                                                                     | New                                                               |
+| --------------------------------------------------------------------------------------- | ----------------------------------------------------------------- |
+| [clickWhenClickable](#clickWhenClickable)                                               | [click](#click)                                                   |
+| [fillFieldWithTextWhenVisible](#fillFieldWithTextWhenVisible)                           | [fillFieldWithText](#fillFieldWithText)                           |
+| [fillInputFieldWithFileWhenPresent](#fillInputFieldWithFileWhenPresent)                 | [uploadFileIntoInputField](#uploadFileIntoInputField)             |
+| [clearFieldWhenVisible](#clearFieldWhenVisible)                                         | [clear](#clear)                                                   |
+| [clearFieldWhenVisibleAndFillItWithText](#clearFieldWhenVisibleAndFillItWithText)       | [clearFieldAndFillItWithText](#clearFieldAndFillItWithText)       |
+| [tapWhenTappable](#tapWhenTappable)                                                     | [tap](#tap)                                                       |
+| [fillFieldWithTextWhenVisibleAndPressEnter](#fillFieldWithTextWhenVisibleAndPressEnter) | [fillFieldWithTextAndPressEnter](#fillFieldWithTextAndPressEnter) |
+| [scrollToElementWhenVisible](#scrollToElementWhenVisible)                               | [scrollToElement](#scrollToElement)                               |
+
+#### What to do?
+
+- Only change the function on **Old** column to the matching function of **New** column, removing the `errorMessage` argument.
+  - All new methods, unlike the old ones, don't use the `errorMessage` argument.
+
+#### How I know that my code is ok?
+
+You will know that you have solved all your problems when don't see any warning on console like this:
```

Comment:
What about: You will know that you have solved all issues when no warnings are shown in the console, like this: ?

Reply:
Same as https://github.com/wlsf82/protractor-helper/pull/43#discussion_r294022938